composer.json

```diff
@@ -39,19 +39,17 @@
     },
     "require-dev": {
         "ext-xdebug": "*",
-        "laminas/laminas-cache": "^3.12.2",
-        "laminas/laminas-cache-storage-adapter-memory": "^2.3",
         "laminas/laminas-coding-standard": "~3.0.1",
         "laminas/laminas-db": "^2.20.0",
         "laminas/laminas-http": "^2.20",
         "laminas/laminas-validator": "^2.64.1",
         "mongodb/mongodb": "~1.20.0",
         "phpunit/phpunit": "^10.5.38",
         "psalm/plugin-phpunit": "^0.19.0",
+        "psr/simple-cache": "^3.0.0",
         "vimeo/psalm": "^5.26.1"
     },
     "suggest": {
-        "laminas/laminas-cache": "Laminas\\Cache component",
```

Comment:
We should suggest `psr/simple-cache-implementation` and/or `psr/cache-implementation` here